### ISSUE TITLE
prettier svelte plugin is required for non-svelte projects

### ISSUE THREAD

**Issue by frattaro** [trivago/prettier-plugin-sort-imports#323]
**Your Environment**
 * **Prettier version**: 3.4.1
 * **node version**: [20.x.x]
 * **package manager**: [npm@latest]
 * **IDE**: [github actions]

**Describe the bug**

running `next lint` (next 14.latest)

```
Cannot find module 'prettier-plugin-svelte'
Require stack:
- /home/runner/work/team/team/node_modules/@trivago/prettier-plugin-sort-imports/lib/src/index.js
Occurred while linting /home/runner/work/team/team/pages/_app.tsx:1
Rule: "prettier/prettier"
```

not using svelte, this is for a react project

**To Reproduce**

**Expected behavior**

shouldn't require a svelte plugin

**Screenshots, code sample, etc**

**Configuration File (cat .prettierrc, prettier.config.js, .prettier.js)**

**Error log**

**Contribute to @trivago/prettier-plugin-sort-imports**

- [ ] I'm willing to fix this bug 🥇

**Comment by chogarcia** [comment]
indeed, downgrading to [v4.3.0](https://github.com/trivago/prettier-plugin-sort-imports/releases/tag/v4.3.0) solves issue

**Comment by byara** [comment]
The issue still happens. Unfortunately, at the moment I do not have any quick way to fix this. Either we let the user pass the plugin they want to the preprocessor or we remove Svelte support for now. The issue comes from the fact that the parsers for Svelte are not provided by Prettier itself (unlike Vue and others) and it makes it hard to support Svelte without having the dependency in your application's dev dependency.

**Comment by rsslldnphy** [comment]
it's really not ideal but could a temporary fix be to move the `prettier-plugin-svelte` from `devDependencies` to `dependencies`? would unnecessarily install it for everyone not using `svelte` but the plugin should still work?

**Comment by frattaro** [comment]
>it's really not ideal but could a temporary fix be to move the prettier-plugin-svelte from devDependencies to dependencies? would unnecessarily install it for everyone not using svelte but the plugin should still work?

agreed. if a dependency has code that's shipped with the package it should be in deps, not devdeps

**Comment by rsslldnphy** [comment]
Feels like it should pretty obviously work, but just to make sure I've forked the repo, made the change, and tested it locally. Can confirm it works, so have pushed the (trivial) PR.

**Comment by byara** [comment]
This issue should be fixed in [v5.2.0](https://github.com/trivago/prettier-plugin-sort-imports/releases/tag/v5.2.0). Please reopen if the issue is still happening.
